### ISSUE TITLE
[prometheus backend] Add prefix for statsd metrics in /api/v1/allmetrics

### ISSUE THREAD

**Issue by Wing924** [firehol/netdata#3515]
If I report `foo:1|c` to statsd, I get 
```ruby
# COMMENT homogeneus chart "statsd_counter_foo.counter", context "foo", family "counters", units "events/s"
# COMMENT netdata_foo_events_persec_average: dimension "counter", value is events/s, gauge, dt 1520404992 to 1520405584 inclusive
netdata_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="counter"} 0.5000000 1520405584000
# COMMENT netdata_foo_events_persec_average: dimension "events", value is events/s, gauge, dt 1520404992 to 1520405584 inclusive
netdata_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="events"} 0.5000000 1520405584000
```
It's hard to find which metrics is reported by statsd.
I suggest add a prefix: `statsd` to all statsd metrics.
Since all tomcat plugin metrics start with `netdata_tomcat_`, all statsd metrics also should start with `netdata_statsd_`.
I know it will break backward compatibility, but it is more nature to handle metrics.
ex:
```bash
# AS IS
netdata_<name>_events_persec_average
# TO BE
netdata_statsd_<name>_events_persec_average
```

**Comment by ktsaou** [comment]
I can't reproduce this. I get:

```
netdata_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="counter"} 0.0000000 1520771269000
netdata_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="events"} 0.0000000 1520771269000
```

So, netdata already does what you propose.
Keep in mind `netdata` is added only when in `netdata.conf` you have (this is also the default):

```
[backend]
	prefix = netdata
```

**Comment by ktsaou** [comment]
oh! sorry. I read it too fast.
You suggest to include `statsd` in these metrics.

**Comment by ktsaou** [comment]
ok, this was a bug.
Private charts had incomplete `context` and `title`. Fixed them with PR #3531

**Comment by ktsaou** [comment]
Now the prometheus metrics are:

```
netdata_statsd_counter_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="counter"} 0.0000000 1520773073000
netdata_statsd_counter_foo_events_persec_average{chart="statsd_counter_foo.counter",family="counters",dimension="events"} 0.0000000 1520773073000
```
